site/src/pages/ManagementSettingsPage/SidebarView.tsx

```diff
@@ -13,18 +13,12 @@ import { type ClassName, useClassName } from "hooks/useClassName";
 import { linkToAuditing, linkToUsers, withFilter } from "modules/navigation";
 
 interface SidebarProps {
-  /**
-   * The active org if an org is being viewed.  If there is no active
-   * organization, assume one of the deployment settings pages are being viewed.
-   */
-  activeOrganization: Organization | undefined;
-  /**
-   * The permissions for the active org or undefined if still fetching (or if
-   * there is no active org).
-   */
-  activeOrgPermissions: AuthorizationResponse | undefined;
-  /** The list of organizations or undefined if still fetching. */
-  organizations: Organization[] | undefined;
+  /** True if a settings page is being viewed. */
+  activeSettings: boolean;
+  /** The active org name, if any.  Overrides activeSettings. */
+  activeOrganizationName: string | undefined;
+  /** Organizations and their permissions or undefined if still fetching. */
+  organizations: [Organization, AuthorizationResponse][] | undefined;
```

Comment:
not a fan of tuples like this. could we do something like...

```ts
interface OrganizationWithPermissions extends Organization {
  permissions: AuthorizationResponse;
}
```

...instead?

Reply:
Yup, works for me!  Yeah the tuple thing feels kinda weird in JavaScript